packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,161 @@
+| showOverlay              | `boolean`                              | `false`                                  | Toggles the visibility of overlay                                                                                                                                                                            |          |
+| overlayColor             | `BrandColorTokens, SurfaceColorTokens` | `surface.background.level1.highContrast` | Changes the color of the overlay, so that consumers can blend the overlay with the background color                                                                                                          |          |
+| navigationButtonPosition | `bottom,side`                          | `bottom`                                 | Sets the position of navigation button                                                                                                                                                                       |          |
+| navigationButtonSpacing  | `SpacingTokens`                        | `spacing.4`                              | Spacing between navigation button and slides when bleed is set to none                                                                                                                                       |          |
+| navigationButtonStyle    | `filled,stroke`                        | `filled`                                 | Styles to be used depending on the emphasis you want to give to the navigation buttons                                                                                                                       |          |
+| indicatorVariant         | `gray,white,blue`                      | `gray`                                   | Variants to be used depending on the background and the emphasis you want to give to the indicator.                                                                                                          |          |
+| onChange                 | `(slideIndex: number) => void`         | `undefined`                              | Callback which gets fired everytime a slide changes, useful for analytic events (only runs on user interaction not on autoPlay)                                                                              |          |
+| accessibilityLabel                 | `string`         | `undefined`                              | Accessibility label for the carousel, this will let screen reader users know what content the carousel holds (eg: "Product carousel")                                                                              |          |
+
+Additionally we will also expose few styled props, namely margin/width etc so that users can add negative margins to create end to end bleed carousel.
```

Comment:
why do we need this now negative margin thing? because we already have bleed capability right?

Reply:
Bleed is for showing partial carousel items, Negative margin will be needed to make the carousel container stretch end to end

```
<body marginX="20px">
  <Carousel marginX="-20px" />
</body>
```